viewer/src/app/feature-service.service.ts

```diff
@@ -96,9 +98,42 @@ export type featureCollectionGeoJSON = {
   numberReturned?: number
 }
 
+export type DataProjectionMapping = {
```

Comment:
Moet dit type niet ProjectionMapping heten? Omdat het zowel DataProjection als een andere (VisualProjection) ondersteund

Reply:
Ja is inderdaad beter